association_measures/measures.py

```diff
@@ -5,149 +5,202 @@
 """
 
 
-import pandas as pd
 import numpy as np
-from .binomial import choose as binomial
+from .frequencies import expected_frequencies, observed_frequencies
 
 
-choose = np.vectorize(binomial) # pylint: disable=invalid-name
+def choose(n, k):
```

Comment:
Can we just fix the choose function as described and not have a second implementation?

See https://github.com/fau-klue/pandas-association-measures/issues/12

Reply:
yes, we can!

and I did.